Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 Compiler Features:
  * Inline Assembly: Issue error when using ``callvalue()`` inside nonpayable function (in the same way that ``msg.value`` already does).
+ * Standard-Json: Support "Yul" as input language.
```

Comment:
We style this usually as "Standard JSON Interface"

Reply:
Fixed.